ts/example/main.ts

```diff
@@ -0,0 +1,24 @@
+import { Scene, Camera, Vector3, SceneFormatIO } from '@computer-graphics-course/scene-format';
+
+(async () => {
+    const scene: Scene = {
+        version: 42,
+        cameras: [<Camera>{
```

Comment:
Very good, never saw this kind of syntax before. I think now if we should mention in docs that we used IIFE and generics inside objects just for the example. As far as i know it is enough to specify only the root type of variable and the other types will be set automatically. But anyway, it is a great approach to show details👍🏼.

Reply:
Yes, this is intentional to make it easier to understand the example.